scbctl/cmd/scans_test.go

```diff
@@ -0,0 +1,85 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"errors"
+	"testing"
+
+	v1 "github.com/secureCodeBox/secureCodeBox/operator/apis/execution/v1"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+type MockClientProvider struct {
+	client.Client
+	namespace string
+	err       error
+}
+
+func (m *MockClientProvider) GetClient(_ *genericclioptions.ConfigFlags) (client.Client, string, error) {
+	return m.Client, m.namespace, m.err
+}
+
+func TestScanCommand(t *testing.T) {
+	scheme := runtime.NewScheme()
+	utilruntime.Must(v1.AddToScheme(scheme))
+
+	testcases := []struct {
```

Comment:
Please also add assertions for the scan parameters. So that we can see that they are passed correctly.

Also a test which basses multiple parameters to the scan would be good too 😊